src/unibit.js

```diff
@@ -352,9 +352,9 @@ module.exports = class Unibit {
         const urlPathParts = _.split(urlPath, '/');
         return _.filter(context.site.pages, page => {
             // find all pages that have same prefix as folder path, but not the root page of that folder, e.g.: {urlPath}/index.html
-            let pageUrl = _.trim(_.get(page, 'url'), '/');
+            let pageUrl = _.trim(_.get(page, 'url'), path.sep);
             pageUrl = prettyUrl(pageUrl);
-            const pageUrlParts = _.split(pageUrl, '/');
+            const pageUrlParts = _.split(pageUrl, path.sep);
```

Comment:
Here we are handling page URL paths. URLs always use forward slash disregards to the underlying system `/`.
On windows system, the `path.sep` will be `\` so the logic will not work for URL paths

Reply:
Here the `path.sep` is not about URL but rather about file routing/parsing.

On windows system, the `page.url` value prints `\\blog\\author\\dianne-ameter`.

- When we try to remove the leading and trailing `/` character(s)  with `_.trim`, nothing happens :
  - `pageUrl` contains `\\blog\\author\\dianne-ameter` instead of `blog\\author\\dianne-ameter`.
- Then we split the url parts by the hardcoded `/`, so we get :
  -  `pageUrlParts = [ '\\blog\\author\\dianne-ameter' ]` instead of `['blog', 'author', 'dianne-ameter']`.

This causes the blog page to not list any blog posts and all subdirectory routes are skipped/broken.

Reply:
Interesting,
Then I guess the root cause of this bug is somewhere before the execution of this code.
The `page.url` should never have OS specific separators and should always have standard URL forward slash - `/`.
Maybe the code that constructs the original `url` property?